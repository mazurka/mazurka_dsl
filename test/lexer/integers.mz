12

123124124123109789879

09123097123