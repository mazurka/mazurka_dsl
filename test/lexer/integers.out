{integer,1,12}.
{integer,3,123124124123109789879}.
{integer,5,9123097123}.